app/static/src/js/components/filtered-fetch-table/filtered-fetch-table.js

```diff
@@ -0,0 +1,194 @@
+AssessFrontend.FilteredFetchTable = function(params) {
```

Comment:
Can we polyfil this? Is the page workable without js?

Reply:
Works without js - polyfill on it